www.foia.gov/_sass/theme/_temp_fixes.scss

```diff
@@ -31,13 +31,17 @@ ul.tabs li {
 }
 
 /* this is necessary because the markup in the HTML doesn't match the markup WDS expects, so it hides the whole thing */
-#selectFY [type=checkbox],
-#selectFY [type=radio] {
-  position: static;
-  left: 0;
-  display: inline-block;
-  width: 10px;
-  height: 1em;
+// TODO https://github.com/18F/beta.foia.gov/issues/178 
+#componentSelectionData, // This markup comes from the Tomcat backend, it is **not** a temporary fix.
```

Comment:
I'm wondering if we should rename this file to `legacy_overrides` or something -- the `temp fix` language isn't the best for where we are now.